README.md

```diff
@@ -1,2 +1,115 @@
 # lsp-deploy
-A place to store the logic and configuration needed to spin up an entire LSP.
+
+This is the repository to store helper scripts and per-environment
+deployment configuration for the services of the Rubin Science
+Platform.
+
+## Background
+
+In order to understand this repository, there's a bit of background
+and layering that needs to be understood.  I will briefly cover each
+layer here, and link to further documentation if you are interested.
+
+The Rubin Science Platform runs on Kubernetes (https://kubernetes.io/).
+Kubernetes provides a way to coordinate running services on multiple
+nodes.  Kubernetes runs a set of docker containers (https://docker.com/),
+and sets up the networking, storage, and configuration of those containers.
+
+Each different service in the Rubin Science Platform is a bit of code that
+eventually gets built and published into a docker container.  The git
+repositories hosting each service generally build their own docker containers
```

Comment:
That sentence reads awkward (i think of something different that a git repo for "host a service")- how about something like "Git repositories for individual services typically have build pipelines resulting in new docker container builds when code changes are merged".

Reply:
Sounds good!